versions.yaml

```diff
@@ -71,7 +71,7 @@ assets:
     description: "Component used to create virtual machines"
 
     cloud_hypervisor:
-      description: "Cloud Hypervisor is an open source Virtual Machine Monitor"
+      description: "Cloud Hypervisor is an open source Virtual Machine Monitor."
```

Comment:
@egernst I dont see any change in the version here.

Reply:
Right, this was to test the changes in packaging.  Which appear to have failed horribly.